src/frontend/Sider/Personoversikt/Dokumentoversikt/LogiskeVedlegg.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { LogiskVedlegg } from '../../../typer/dokument';
+
+const LogiskeVedlegg: React.FC<{ logiskeVedlegg: LogiskVedlegg[] }> = ({ logiskeVedlegg }) => {
+    return logiskeVedlegg.map((logiskVedlegg) => (
+        <div key={`${logiskVedlegg.tittel}-${logiskVedlegg.logiskVedleggId}`}>
```

Comment:
Tror det er tilstrekkelig med `logiskVedleggId` som key.
```suggestion
        <div key={logiskVedlegg.logiskVedleggId}>
```